src/common/error.ts

```diff
@@ -1,177 +1,86 @@
-import { getLogger } from "../common/logger";
-
-const codedErrorSymbol = Symbol("Error code property");
-
-const { error: errorLogger } = getLogger("src/common/error");
+import { APIGatewayProxyResult } from "aws-lambda";
 
 /* eslint-disable no-unused-vars */
 /* eslint-disable no-shadow */
-export enum ErrorCodes {
-  // 4xxx invalid requests error
-  MISMATCHING_PARTICULARS = 4001,
-  UNRECOGNISED_CLINIC = 4002,
-  TEST_RESULTS_EXPIRED = 4003,
-  DOCUMENT_INVALID = 4004,
-  DATA_INVALID = 4005,
-  FILE_TYPE_INVALID = 4006,
-  FILE_INVALID = 4007,
-
-  // 5xxx internal server error
-  EU_QR_ERROR = 5001,
+export enum ErrorType {
+  // 4xx invalid requests error
+  INVALID_REQUEST_PAYLOAD = 400,
+  INVALID_CERTIFICATE = 400,
+  INVALID_DOCUMENT = 400,
+  INVALID_SCHEMA = 400,
+  INVALID_HEALTHCERT_TYPE = 400,
+  INVALID_LOGO = 400,
+  INVALID_LOGO_SIZE = 400,
```

Comment:
Should we merge these together? Then rely on `e.message` for the difference.

Reply:
I'll merge both into `INVALID_LOGO ` ! Agree to rely on `e.message` for this